install/kubernetes/cilium/values.yaml

```diff
@@ -1777,6 +1778,30 @@ tls:
     # -- Generated certificates validity duration in days. This will be used for auto generated CA.
     certValidityDuration: 1095
 
+  # -- Configure the CA trust bundle used for the validation of the certificates
+  # leveraged by hubble and clustermesh. When enabled, it overrides the content of the
+  # 'ca.crt' field of the respective certificates, allowing for CA rotation with no down-time.
+  caBundle:
```

Comment:
Could this be part of the `ca` field above instead?

Reply:
I had the same doubt when adding this field. Personally, I feel it is a bit better to keep them disjoint to highlight that they are not strictly related, since you might want to specify the bundle but not the CA info (e.g., when using cert-manager). I don't have a strong opinion though.

Reply:
what happens when both `ca` and `caBundle` are specified?

Reply:
These two fields are independent. Essentially, `ca` is used to specify the CA for certificate generation through helm or the cronJob, while the `caBundle`, if set, overrides the `ca.crt` entry part of the certificate secret (which would be `ca.tls` if generated through helm/cronJob, but may be different), and it represents the set of trusted root certificates for the different cilium components. It mainly enables seamless CA rotation, since you can specify multiple trusted certificates at the same time. 

For instance, you would configure `caBundle` only when relying on cert-manager to generate the certificates, `ca` only if you are only interested in certificate generation through helm/cronJob, or both in case you want both certificate generation and the support for CA rotation with no downtime.